### PR TITLE
🐛 FIX: horizontal scrolling on mobile due to tab accessibility navigation 'skip to content' positioned at a fixed position

### PR DESCRIPTION
**Pull request by violoncelloCH**

### CODE REVIEW

File: core/css/mobile.scss [hunk 1]
```diff
@@ -32,6 +32,10 @@
 		margin-left: 0;
 	}
 
+	.skip-navigation.skip-content {
+		left: 3px;
+		margin-left: 0px;
```
Comment (jancborchardt) [inline core/css/mobile.scss]:
```suggestion
		margin-left: 0;
```

0 does not need a unit. :) Just a detail though, great work!

Comment (violoncelloCH) [reply core/css/mobile.scss]: ah, yeah, obviously... let's change that...

**Review by jancborchardt — APPROVED** [review]
Looks good! :)

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**